starter-code/index.html

```diff
@@ -10,15 +10,19 @@
     <link rel="stylesheet" href="styles/layout.css">
     <link rel="stylesheet" href="styles/modules.css">
     <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+    <script id="blog-articles" type="text/x-handlebars-template">
+      <article>
+        <header>
+          <h1>{{title}}</h1>
+          <div class="byline">
+              By <address><a href="">{{author}}</a></address>
```

Comment:
Now when you click on author's name, you don't have the link D:
That's because we removed the line
`$newArticle.find('.byline a').attr('href', this.authorUrl);` in `article.js`
So you probably want to add that over here in `href=""`.